tests/x11/vlc.pm

```diff
@@ -26,10 +26,16 @@ sub run {
     x11_start_program('vlc --no-autoscale', target_match => 'vlc-first-time-wizard');
     assert_and_click "vlc-first-time-wizard";
     assert_screen "vlc-main-window";
-    send_key "ctrl-l";
-    assert_and_click "vlc-playlist-empty";
-    send_key "ctrl-n";
-    assert_screen "vlc-network-window";
+    my $counter = 3;
+    while ($counter--) {
+        send_key "ctrl-l";
+        last if (check_screen "vlc-playlist-empty", 60);
+    }
+    my $counter = 3;
+    while ($counter--) {
+        send_key "ctrl-n";
+        last if (check_screen "vlc-network-window", 60);
+    }
```

Comment:
This is rather hard to maintain code. If you think you really need this you can try `send_key_until_needlematch`.